### ISSUE TITLE
Space Measurement Ideas

### ISSUE THREAD

**Issue by RonaldThomas** [TinkerMill/TinkerSpaceCommand#2]
I've created the list below as a quick braindump of all the things we might want to measure at TinkerMill. For many of the items below, the ability to record or alarm events triggered by state changes would be hugely helpful.

- Temperature
- Humidity
- Motion
- Light Level
- Light Switch Status
- Doors and Gate
  - Gate Open Status
  - Door Lock Status
  - Door Open Status
- Mains Power Monitoring
  - Voltage
  - Current Draw
    - By Facility
    - By Panel
    - By Circuit
    - By Outlet
    - By Tool
- Natural Gas
  - Valve Status
- Gas Cylinders
  - Cylinder Identification
  - Weight
  - Valve Status
-Shop Air
  - Current Pressure
  - Compressor Power Status
  - Valve Status
  - Compressor Duty Cycle
- Occupancy 
  - Count of Members Present
  - Count of Visitors Present
  - Identification of Members Present

**Comment by kmhughes** [comment]
Awesome!

So we can start hitting the system with some data, Steve and I want to start building some of the sensors so we can really test the model and software out.

Here are the spaces we are thinking about. Please add any additional ones.

Main bay
wood shop
Storage room next to woodshop
old metal shop
new metal shop
classroom
IT room
ham shop
conference room
room where offices and bathrooms are
ceramics
back bay (outside with blacksmithing)

**Comment by RonaldThomas** [comment]
Do we want to add any measurement for the outdoors? We have a nearly complete Sparkfun Photon weather system (https://www.sparkfun.com/products/13674) that just needs to be finished and installed.

There are three additional rooms in the modular building we might consider:
- Recording studio
- Middle room
- Pick and place room

**Comment by kmhughes** [comment]
Yes, we will include the outdoors measurements as well, and, as per our conversation, push the weather data at Weather Underground. Thanks for mentioning that.
